mod/01_rating/spec/set/right/downvotee_search_spec.rb

```diff
@@ -24,7 +24,7 @@
           vcc2.save!
         end
         metric_downvotee_search_card = Card.fetch "#{apple.name}+metric+downvotee search"
-        result = metric_downvotee_search_card.item_cards
+        result = metric_downvotee_search_card.format.get_search_result
```

Comment:
that makes sense, but I think we might be able to organize this a bit more clearly:
1. get_search_result should have a clearer name, and
2. it should probably be a method on the model, not the format.  It's much more natural for the format object to call this (it already has a card object) rather than have the model create a format object just for this purpose (card.format creates a new object)